### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by abelzer212**
## Purpose

Modernize and flesh out our current application documentation

#### Linked Issues to Close

https://qmacbis.atlassian.net/browse/OY2-15019

## Approach

_How does this change address the issue?_

## Learning

_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Assorted Notes/Considerations

_List any other information that you think is important... a post-merge activity, someone to notify, etc._

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,165 +1,669 @@
...
+   URL=http://localhost/3000
+   SLS_INTERACTIVE_SETUP_ENABLE=1
+   ```
+
+1. Set up the UI-SRC ENV.
+
+   1. Navigate to `/services/ui-src/`.
+   1. Make a new file: `.env`
+   1. Copy the contents of `.env_example` into `.env`
+      1. If you want to connect to real resources for the branch you can tweak these values with the resource values found in AWS.
```
Comment (jphechter) [inline README.md:298]: Both `.evn` files will get automatically generated with the contents of `.env_example` which are stored in source control. This behavior can be confirmed by deleting both `.env` files and running the `dev` script.

I think there's value in explaining the content of these files, but I think we could drop that in later.

File: README.md [hunk 2]
```diff
@@ -1,165 +1,834 @@
...
+SHIFT + CMD/CTRL + V
+-->
 
-## Release
+# Quality Measure Reporting (QMR)
 
-Our product is promoted through branches. Master is merged to val to affect a master release, and val is merged to production to affect a production release. Please use the buttons below to promote/release code to higher environments.<br />
+# cms-mdct-qmr ![Build](https://github.com/CMSgov/cms-mdct-qmr/workflows/Deploy/badge.svg?branch=master) [![latest release](https://img.shields.io/github/release/cmsgov/cms-mdct-qmr.svg)](https://github.com/cmsgov/cms-mdct-qmr/releases/latest) [![Maintainability](https://api.codeclimate.com/v1/badges/1449ad929006f559756b/maintainability)](https://codeclimate.com/github/CMSgov/cms-mdct-qmr/maintainability) [![Dependabot](https://badgen.net/badge/Dependabot/enabled/green?icon=dependabot)](https://dependabot.com/) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier) [![Test Coverage](https://api.codeclimate.com/v1/badges/f5b10ae50ca1effedcd3/test_coverage)](https://codeclimate.com/repos/60fae00673444f5bad001bf9/test_coverage)
```
Comment (TheDanMiller) [inline README.md:9]: image
Maintainability still looks busted when I look at it

File: README.md [hunk 3]
```diff
@@ -1,165 +1,834 @@
...
+
+# Table of Contents
+
+- [Quality Measure Reporting (QMR)](#quality-measure-reporting-qmr)
+- [cms-mdct-qmr !Build [![latest release](https://img.shields.io/github/release/cmsgov/cms-mdct-qmr.svg)](https://github.com/cmsgov/cms-mdct-qmr/releases/latest) [![Maintainability](https://api.codeclimate.com/v1/badges/1449ad929006f559756b/maintainability)](https://codeclimate.com/github/CMSgov/cms-mdct-qmr/maintainability) [![Dependabot](https://badgen.net/badge/Dependabot/enabled/green?icon=dependabot)](https://dependabot.com/) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier) [![Test Coverage](https://api.codeclimate.com/v1/badges/f5b10ae50ca1effedcd3/test_coverage)](https://codeclimate.com/repos/60fae00673444f5bad001bf9/test_coverage)](#cms-mdct-qmr------)
```
Comment (TheDanMiller) [inline README.md:30]: image
Maintainability again appears busted here

File: README.md [hunk 4]
```diff
@@ -1,165 +1,834 @@
...
+| AWS_Splunk_User           | User Level Access for AWS Splunk Application                                                                                                                                                                             |
+| CTR_VPN_3ZWINMF_Exception | One of 2 codes required for VPN access. Has a 90 day auto-expire. Needed while getting approved for a PIV. See [VPN section](#vpn) for more information.                                                                 |
+| CTR_VPN_PIV_3ZWINMF       | Contractor VPN_PIV access to 3-Zone URL's, Windows servers & Mainframes. See [VPN section](#vpn) for more information.                                                                                                   |
+| CHIP_D_User               | CHIP-State Childrens Health Ins Prog Enrollmnt Data Sys Dev Access                                                                                                                                                       |
```
Comment (TheDanMiller) [inline README.md:129]: Why do we need the CHIP codes? Is there an integration here that we need these codes for specific to QMR?

File: README.md [hunk 5]
```diff
@@ -1,165 +1,834 @@
...
+yarn test
+```
+
+The `Cypress` application will kick off, where you can find a list of all the available E2E tests.
 
-- to run the dynamodb gui, run `DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin` in a new terminal window
+To run an invidiual Child Measure test, you first need to create these measures by running the `create_delete_child.spec.ts` test.
 
-## Testing
+Similarly, to run an individual Health Home Measure test, you first need to create these measures by running the `create_delete_healthhome.spec.ts` test.
```
Comment (TheDanMiller) [inline README.md:450]: Does this stay true with the upcming shift in testing -- sorry if this is not a great question

File: README.md [hunk 6]
```diff
@@ -1,165 +1,834 @@
...
+Code Climate is a code analytics tool that measures unit test coverage, code complexity, and code readability.
 
-[See here](./tests/cypress/README.md)
+It is connected to the repository and can be reached [here](https://codeclimate.com/dashboard)
+
+When logging in, choose "Login with GitHub" to connect to the appropriate repos.
+
+### Configuration
+
+The Code Climate config file is at the top level of the project and called `.codeclimate.yml`. This is similar to a gitignore file but it determines what parts of the repo to measure for code coverage.
```
Comment (TheDanMiller) [inline README.md:504]: I am pretty sure this is incorrect. The codeclimate.yml file excludes sections from codesmell and maintainability checks. The coverage reporting is driven from the collectCoverageFrom in the package.json files.

File: README.md [hunk 7]
```diff
@@ -170,15 +839,39 @@ to make in order to get that year working.
...
+   Before
+
+   ![Before](./.images/beforeImportUpdate.png?raw=true)
+
+   After
 
-## Contributing / To-Do
+   ![After](./.images/afterImportUpdate.png?raw=true)
+
+5. Copy over the `/globalValidations`, `/CommonQuestions`, and `/Qualifiers` directories to the latest year
```
Comment (TheDanMiller) [inline README.md:856]: Do the questions change YoY. This seems like a hefty amount of duplication. This does not impact the readme, just more of a general question

**Review by TheDanMiller — APPROVED** [review]
Left some comments -- address as you will

File: README.md (hunk 5 above)
Comment (abelzer212) [reply README.md:450]: Yes for now, if the process changes we can change the documentation.

File: README.md (hunk 7 above)
Comment (abelzer212) [reply README.md:856]: The questions do change year over year, (or that's the specifications we received) There is a possibility that questions can change entirely or just in phrasing, but we need to maintain up to 5 years worth of data. I think going forward the previous years will be in a read only view. But yeah right now there's a lot of duplication to try to anticipate large changes to forms.

File: README.md (hunk 2 above)
Comment (abelzer212) [reply README.md]: I'm guessing that link doesn't exist anymore. It has been removed

**Pull request merged**